src/sass/layout/_products.scss

```diff
@@ -227,4 +256,12 @@ width: 100%;
         padding-right: 44px;
         padding-bottom: 62px;
     }
+
+    .button-products {
+
+        &--hero {
+            width: 26px;
+            height: 26px;
+        }
+    }
 }
```

Comment:
to zamkniecie nie nalezy do header-h2?

Reply:
nie, u mnie wszystko jest ok pozamykane, ostatni nawias jest od media queries

Reply:
to media queries musza sie zamknac